addons/packages/calico/3.22.1/package.yaml

```diff
@@ -51,7 +46,7 @@ spec:
       syncPeriod: 5m
       fetch:
       - imgpkgBundle:
-          image: projects.registry.vmware.com/tce/calico@sha256:31c240996c50b1fd06339987465a95b01a982050fff1ecd0262e8ae1e50206e4
+          image: index.docker.io/licrui/tce-calico@sha256:a4ccb6a2a6871f2b1d24a04416f0e99d6bb1f295c6c0eeee0546992470b57840
```

Comment:
Can you push this to `projects.registry.vmware.com/tce`?

Reply:
@12345lcr Sorry, just re-read your comment above. You can use the `push-package` make task in the TCE repo, just use a tag like `3.22.1-2`. We've enabled immutable tags, so we can no longer re-use tags.

```
make push-package PACKAGE=calico VERSION=3.22.1 TAG=3.22.1-2
```